libpod/networking_linux.go

```diff
@@ -282,6 +283,7 @@ func (r *Runtime) setupSlirp4netns(ctr *Container) error {
 	enableIPv6 := false
 	outboundAddr := ""
 	outboundAddr6 := ""
+	mtu := -1
```

Comment:
Should we keep the default of 65536 if unset? @giuseppe @AkihiroSuda

Reply:
Sorry, 65520

Reply:
Yes

Reply:
This seems wrong to me, why would podman force 65520 if my network is configured differently? Doesn't slirp honor the host mtu? After testing on multiple machines and network adapters, 65520 was always problematic vs the default mtu.

Reply:
In general a higher MTU means higher throughput. However the MTU should be the same on each interface that the packet passes through. Physical NICs typically have an MTU of 1500. Some high-end devices operate with an MTU of 9000. If a packet with a higher MTU has to go through this NIC, it is repackaged into smaller packets. This causes the big performance drop you see. So if you are only communicating between the host and the container, you want an MTU that high. https://github.com/rootless-containers/slirp4netns#iperf3-netns---host That's why I think this MTU was chosen, although I don't understand why it is 65520 and not 65536.

Reply:
Yes, the loopback device mtu is set at 65536 by default. But my point stands, podman forces a specific scenario based value which I don't think is sane.

I will, however, change the code if it's the way the project wants to go.

Reply:
`slirp4netns` help:

-m, --mtu=MTU            specify MTU (default=1500, max=65521)

I guess it's the almost max?

Reply:
Changing the default can be discussed in a separate issue/PR with actual benchmark data.

Reply:
Actually 65520 is known not to be optimal on recent kernel with the default sysctl values: https://github.com/rootless-containers/slirp4netns/issues/128

Maybe we can adjust the default dynamically based on sysctl values. (Or rather adjust sysctl dynamically)

Reply:
Yes, anyways, I have made 65520 the default again.